ExampleApp/ExampleApp.xcodeproj/project.pbxproj

```diff
@@ -619,21 +619,21 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		8D72C5882539C65000456D1A /* XCRemoteSwiftPackageReference "RxBluetoothKit" */ = {
+		D3EAD5002A374639004DF276 /* XCRemoteSwiftPackageReference "RxBluetoothKit" */ = {
 			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/Polidea/RxBluetoothKit";
+			repositoryURL = "https://github.com/Airthings/RxBluetoothKit.git";
 			requirement = {
-				kind = upToNextMajorVersion;
-				minimumVersion = 6.0.0;
+				branch = "update-rxswift-to-6.5";
```

Comment:
I think this reference should be updated before merge

Reply:
I'm looking into it

Reply:
This is a cat and mouse case. Have to first merge and then change the branch. Otherwise, have to revert the Example Project code and create a new PR. Too much work. I will create PR to change the branch immediately.